composer.json

```diff
@@ -31,22 +31,13 @@
         }
     },
     "scripts": {
-        "increase-severity": "sed -i.bak 's/<severity>0<\\/severity>/<!--<severity>0<\\/severity>-->/' CakePHP/ruleset.xml",
-        "reset-ruleset": [
-            "sed -i.bak 's/<!--<severity>0<\\/severity>-->/<severity>0<\\/severity>/' CakePHP/ruleset.xml",
-            "rm -f CakePHP/ruleset.xml.bak"
-        ],
         "add-standard" : "phpcs --config-set installed_paths $(pwd)",
         "test": [
           "@add-standard",
-          "@increase-severity",
-          "phpunit",
-          "@reset-ruleset"
+          "phpunit"
         ],
-        "cs-check": "phpcs --colors -p -s --exclude=SlevomatCodingStandard.TypeHints.DeclareStrictTypes --standard=CakePHP CakePHP/Sniffs/",
-        "cs-fix": "phpcbf --colors -p --exclude=SlevomatCodingStandard.TypeHints.DeclareStrictTypes --standard=CakePHP CakePHP/Sniffs/",
-        "lowest": " validate-prefer-lowest",
-        "lowest-setup": "composer update --prefer-lowest --prefer-stable --prefer-dist --no-interaction && cp composer.json composer.backup && composer require --dev dereuromark/composer-prefer-lowest && mv composer.backup composer.json",
```

Comment:
this should be kept for bc check!

Reply:
Sure, if you really want it.

Reply:
it is important to know when we rely on the wrong sniffer repo version, which is not compatible anymore with what we use.